preprocess_era5_modellevels.py

```diff
@@ -86,48 +83,78 @@ def load_modellevel_data(variable, date):
     evap *= a_gridcell  # m3
     precip *= a_gridcell  # m3
 
+    # Transfer negative (originally positive) values of evap to precip
+    precip = np.maximum(precip, 0) + np.maximum(evap, 0)
+    # Change sign convention to all positive,
+    evap = np.abs(np.minimum(evap, 0))
+    
     # Convert model levels to pressure values
     sp_modellevels2 = sp.expand_dims({'lev':modellevels},axis=1)
     p_modellevels = a_edge + b_edge*sp_modellevels2 # in Pa
 
     #calculate the difference between the pressure levels
     dp_modellevels = p_modellevels.diff(dim="lev") # in Pa
 
-    # Determine the fluxes and states
-    fa_e = u * q * dp_modellevels / g  # eastward atmospheric moisture flux
-    fa_n = v * q * dp_modellevels / g  # northward atmospheric moisture flux
-    cwv = q * dp_modellevels / g * a_gridcell[np.newaxis,np.newaxis,:] / density_water  # column water vapor (m3)
-
     # Split in 2 layers 
     # To do: Check if this is a reasonable choice
     boundary = 111 # set boundary model level 111
     
     # Integrate fluxes and states to upper and lower layer
     lower_layer = dp_modellevels.lev > boundary 
     upper_layer = ~lower_layer
-
-    # Integrate fluxes and state
-    fa_e_lower = fa_e.where(lower_layer).sum(dim="lev")
-    fa_n_lower = fa_n.where(lower_layer).sum(dim="lev")
-    w_lower = cwv.where(lower_layer).sum(dim="lev")
-
-    fa_e_upper = fa_e.where(upper_layer).sum(dim="lev")
-    fa_n_upper = fa_n.where(upper_layer).sum(dim="lev")
-    w_upper = cwv.where(upper_layer).sum(dim="lev")
+        
+    if config['vertical_integral_available'] == True:
+        #load total water column
+        tcw = load_surface_data("tcw", date) # kg/m2
+        #vieclw = load_surface_data("vieclw", date) # kg m-1 s-1
```

Comment:
Are these variables still necessary?

Reply:
No not in the way we do it now, I deleted them and will commit after